### DISCUSSION TITLE
google colab

### DISCUSSION THREAD

**Discussion by seungyong0313** [zhiqwang/yolov5-rt-stack#296] [General]
I tried following yolort's tutorial, but I get a lot of errors. Do you have any plans to provide tutorials in the colab environment?

**Comment by zhiqwang** [comment]
Hi @seungyong0313 ,

Thanks for reporting this. This is in our plan, currently the tutorial needs to be run on the main branch, I guess the reason for the error is probably because of this, you can install the developer version like this, 
```
pip install -U 'git+https://github.com/zhiqwang/yolov5-rt-stack.git'
```
 You can send us the error message marked as `bug` if there is still an error, and we will try to solve these.

And we will release version 0.6.0 soon, and deploy the tutorial with a Colab environment.
